edx-platform/pearson-vue-theme/lms/templates/header/header.html

```diff
@@ -1,37 +1,125 @@
 ## mako
 
-<%page expression_filter="h"/>
-
-<%namespace name='static' file='/static_content.html'/>
+<%page expression_filter="h" args="online_help_token, use_cookie_banner=False"/>
 
+<%namespace name='static' file='../static_content.html'/>
+<%namespace file='../main.html' import="login_query"/>
 <%!
 from django.urls import reverse
 from django.utils.translation import ugettext as _
-from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+import waffle
+
+from lms.djangoapps.ccx.overrides import get_current_ccx
+from openedx.core.djangolib.markup import HTML, Text
 
 # App that handles subdomain specific branding
 from branding import api as branding_api
+from openedx.core.djangoapps.lang_pref.api import header_language_selector_is_enabled, released_languages
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+%>
+<%
+theme = configuration_helpers.get_value('theme', '')
```

Comment:
I think this variable is no longer used on this template. @luismorenolopera

Reply:
removed